juriscraper/opinions/united_states/state/neb.py

```diff
@@ -8,36 +9,65 @@ def __init__(self, *args, **kwargs):
         self.url = (
             "https://supremecourt.nebraska.gov/courts/supreme-court/opinions"
         )
-        self.status = "Published"
+
+    def _return_response_text_object(self):
+        """Remove faulty URLs from HTML
+
+        Override _return_response_text_object in Abstract Site to remove any
+        url to [site:url-brief] which causes lxml/ipaddress to explode.
+
+        This bug only exists in 3.11 and 3.12
+
+        <a href="https://[site:url-brief]/node/19063" ... >District Map</a>
+
+        :return: The cleaned html tree of the site
+        """
+        if self.request["response"]:
+            payload = self.request["response"].text
+            text = self._clean_text(payload)
+            html_tree = self._make_html_tree(text)
+
+            for tag in html_tree.xpath(
+                "//a[contains(@href, 'site:url-brief')]"
+            ):
+                parent = tag.getparent()
+                if parent is not None:
+                    parent.remove(tag)
+
+            if hasattr(html_tree, "rewrite_links"):
+                html_tree.rewrite_links(
+                    fix_links_in_lxml_tree, base_href=self.request["url"]
+                )
+            return html_tree
 
     def _process_html(self):
-        path = {
-            "group": '//div[contains(@class, "view-grouping")]',
-            "date": './/span[contains(@class, "date-display-single")]',
-            "row": ".//tbody/tr",
-        }
-        for group in self.html.xpath(path["group"]):
-            date_element = group.xpath(path["date"])
-            if not date_element:
+        for table in self.html.xpath(".//table"):
+            date_tags = table.xpath("preceding::time[1]/text()")
+            if not date_tags:
                 continue
-            date = date_element[0].text_content()
-            for row in group.xpath(path["row"]):
-                url = row.xpath(".//td[3]//a/@href")
-
-                # skip rows without url in third cell
-                if not url:
-                    continue
+            date = date_tags[0]
 
-                docket = row.xpath(".//td[1]")[0].text_content().strip()
-                if docket == "A-XX-XXXX":
+            for row in table.xpath(".//tr/td/.."):
+                c1, c2, c3 = row.xpath(".//td")
```

Comment:
This is throwing an error
```
Traceback (most recent call last):
  File "sample_caller.py", line 263, in main
    site.parse()
  File "/juriscraper/juriscraper/AbstractSite.py", line 140, in parse
    self._process_html()
  File "/juriscraper/juriscraper/opinions/united_states/state/neb.py", line 52, in _process_html
    docket = c1.xpath(".//span/text()")[0].strip()
IndexError: list index out of range
```

I think the page we see is not the same that is sent from the server
If you open the file sent, there is no span tag
You can this code to see it
```
from lxml.html import fromstring, open_in_browser
open_in_browser(fromstring(self.request['response'].content))
```
![image](https://github.com/freelawproject/juriscraper/assets/14970769/cfbf1d69-90fc-4fc0-bb4e-f15c197a91a8)


As this selector is failing, others may fail too

Reply:
good catch -